Projects/Features/Scene/HomeScene/HomeView.swift

```diff
@@ -17,7 +18,41 @@ public struct HomeView: View {
     }
     
     public var body: some View {
-        Text("Home View")
-            .onAppear { store.send(.onAppear) }
+        VStack {
+            LKTopLogoBar {
+                store.send(.addLinkButtonTapped)
+            }
+            
+            ScrollView {
+                Color.pink
+                    .frame(height: 306)
+                
+                LazyVStack(pinnedViews: .sectionHeaders) {
+                    Section {
+                        LazyVStack(spacing: 0) {
+                            ForEach(0..<store.cards.count, id: \.self) { index in
```

Comment:
```suggestion
                            ForEach(store.cards.indices, id: \.self) { index in
```
indicies를 사용하는 방법도 있어! 참고혀~